e2e/cypress/integration/enterprise/ldap/ldap_group_sync_spec.js

```diff
@@ -339,5 +339,165 @@ context('ldap', () => {
+                // * Should should no results after looking for channel
+                cy.get('.no-results__title').should('be.visible').and('contain.text', 'No results for');
+            });
+        });
+
+        it('MM-T2641 - Channel appears in More... under Public Channels before conversion but not after', () => {
+            // # Create new test channel that is public
+            cy.apiCreateChannel(
+                testTeam.id,
+                'a-channel-im-not-apart-off',
+                'Public channel',
+                'O',
+            ).then((res) => {
+                const publicChannel = res.body;
+                cy.apiLogin(testUser);
+
+                // # Click more under public channels
+                cy.visit(`/${testTeam.name}/channels/off-topic`);
+                cy.get('#sidebarPublicChannelsMore').click();
+
+                // * Search public channel and ensure it appears in the list
+                cy.get('#searchChannelsTextbox').type(`${publicChannel.display_name}`);
+                cy.get('#moreChannelsList').should('include.text', publicChannel.display_name);
+
+                // # login as a admin and revert to private channel
+                cy.apiAdminLogin();
+                cy.apiPatchChannelPrivacy(publicChannel.id, 'P');
+
+                // # Login as a nromal user
+                cy.apiLogin(testUser);
+
+                // # Click more under public channels
+                cy.visit(`/${testTeam.name}/channels/off-topic`);
+                cy.get('#sidebarPublicChannelsMore').click();
+
+                // * Search private channel name and make sure it isn't there in public channel directory
+                cy.get('#searchChannelsTextbox').type(`${publicChannel.display_name}`);
+                cy.get('#moreChannelsList').should('include.text', 'No more channels to join');
+            });
+        });
+
+        it('MM-T2642 - Channel appears in Integrations options before conversion but not after', () => {
+            cy.visit(`/${testTeam.name}/channels/${testChannel.name}`);
+
+            // # Go to integrations
+            cy.visit(`/${testTeam.name}/integrations`);
+
+            // # Go to outgoing webhooks and then add out going web hooks page
+            cy.get('#outgoingWebhooks', {timeout: TIMEOUTS.ONE_MIN}).should('be.visible').click();
+            cy.get('#addOutgoingWebhook', {timeout: TIMEOUTS.ONE_MIN}).should('be.visible').click();
+
+            // * In the channel select options, ensure our display name appears
+            cy.get('#channelSelect').children().should('contain.text', testChannel.display_name);
+
+            // # Make channel private
+            cy.apiPatchChannelPrivacy(testChannel.id, 'P');
+
+            // # Go to integrations
+            cy.visit(`/${testTeam.name}/integrations`);
+
+            // # Go to outgoing webhooks and then add out going web hooks page
+            cy.get('#outgoingWebhooks').should('be.visible').click();
```

Comment:
This was failing for me locally without the timeout,

```suggestion
            cy.get('#outgoingWebhooks', {timeout: TIMEOUTS.ONE_MIN}).should('be.visible').click();
```

Reply:
Done!